src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -737,6 +743,41 @@ private boolean addGitHubCommitter(Channel channel, User sender, String collabor
         return result;
     }
 
+    private boolean removeGitHubCommitter(Channel channel, User sender, String collaborator, String justForThisRepo) {
+        boolean result = false;
+        if (!isSenderAuthorized(channel,sender)) {
+            insufficientPermissionError(channel);
+            return false;
+        }
+        OutputChannel out = channel.send();
+        try {
+            if (justForThisRepo == null) {
+                // legacy command
+                out.message("I'm not longer managing the Everyone team. Please add committers to specific repos.");
+                return false;
+            }
```

Comment:
not needed in new commands / could be binned I think. or at least reword to something more sensible

Reply:
Removed it here, the IRC bot docs don't lose a word about this functionality, I guess it's time to move on and remove it here.